HelpSource/Classes/Integer.schelp

```diff
@@ -1,12 +1,14 @@
 class:: Integer
-summary:: Integer number
+summary:: 32-bit Integer number
 categories:: Math
 
 description::
-A 32 bit integer. Integer inherits most of its behaviour from its superclass.
+A 32-bit integer. Integer inherits most of its behaviour from its superclass.
 
 note::
-A 32 bit signed integer can represent values in the range -2147483648 to 2147483647. Adding up further returns in a wrapped result, so that and 2147483647+1 = -2147483648. For a larger range, one can use link::Classes/Float::, which is 64 bit and supports many (but not all) numerical methods that int does.
+Integer can represent values in the range -2147483648 to 2147483647.
+
+The overflow behavior of an Integer is undefined. Consider using the 64-bit link::Classes/Float:: for larger numbers (from code::±2^53::).
```

Comment:
Unless I'm missing something, "from 2^53" should be -> "up to 2^53" ?

If using exponent here, I'd also include it in the line about the Integer range:

> Integer can represent values in the range -2147483648 to 2147483647 (±2^31).

Though technically that's -2^31 to +2^31-1, so I'm not sure how to handle this both elegantly and accurately.